packages/svelte/src/internal/client/dom/elements/bindings/media.js

```diff
@@ -138,56 +138,28 @@ export function bind_playback_rate(media, get, set = get) {
  * @param {(paused: boolean) => void} set
  */
 export function bind_paused(media, get, set = get) {
-	var mounted = hydrating;
 	var paused = get();
 
-	var callback = () => {
+	var update = () => {
 		if (paused !== media.paused) {
-			paused = media.paused;
 			set((paused = media.paused));
 		}
 	};
 
-	if (paused == null) {
-		callback();
-	}
-
-	// Defer listening if not mounted yet so that the first canplay event doesn't cause a potentially wrong update
-	if (mounted) {
-		// If someone switches the src while media is playing, the player will pause.
-		// Listen to the canplay event to get notified of this situation.
-		listen(media, ['play', 'pause', 'canplay'], callback, false);
-	}
+	// If someone switches the src while media is playing, the player will pause.
+	// Listen to the canplay event to get notified of this situation.
```

Comment:
Should this be the expected behavior? This can be really annoying to deal with.
For context, that is the situation: https://svelte-5-preview.vercel.app/#H4sIAAAAAAAACqVSwW7bMAz9FUKXJoARNzns4MUBti_oYbd5KByZtoXKpCFSHQrD_z4oStd1wbBDT4QeH98jHrWY3nkUU31fDLUTmsp8mWdTGH2Z00Oe0SuawgjHYBNyFBvcrKeGGvWooG5CqOH-8ysgwUINFL3_Dc1tFOyghr71ggmmRi2TKMy-fYEaNiTBbqE-wZJ6jb6TbfQqKsFegVtJXf_WbWPneJ_U_1ROnc3dqDpLVZatVccku4F58LizPJXCkTopn_dl69swSS6P1rN92vEw3G3_7Xb4uNs5Dh4fNRK-MzuWb8HT8WIGZ0ddZWMISPrNTVgvKbY14zmhesl1hUWCXU_H8jKaVc5RlQmYrHf2qV7eQlsv931Ix8lW-7RB5v9n9nAze7iZDaeGLgQdEXoXRDO1gJY6OGPPAcEpIHVS5D-SmKwjBmBC2MjMzmOoEuun8x6I8xm2r_pfWcfrAmMr8AkELVMnDZnCTNy53mFnKg0R1x_rL2WeCyQJAwAA

Reply:
And with the canplay handler, it needs a setTimeout to work via the binded pause

https://svelte-5-preview.vercel.app/#H4sIAAAAAAAACqVSwW7bMAz9FUIYkAQw4iaHHdw4wPYFO-y2DIUiM4kQmTREukNg-N8HWUmztih62EEQ9Pj4HilyMAcfUEz1azBkWzSV-dZ1pjB66dJDnjEomsII99ElZCMu-k63O9qpbzuOCgOod2cY4RC5hVnOmSVCOgEV1LcINXwRtYrzh8XjLSDR3XHqQ7iHOtsLNvfowQbBKUw7dUyi0AV7gRrmJNEtoN7CMJnq1e7hMT-zSSJdgRfpSXMCx7e6tm88r5L6v8opMp-dVDupytI69UyyPDIfAy4dt6VwT42Uz6vSBhtbydeTC-zOSz4eZ4uP3db_77bvjwGftCf8wIzJWbr-m5ULudeOgvrTt8i9zl_hb_8so2MBq5vFprwvBm2mfmDvqalcHyPSJFsPaTJjxrNgPeR7hEGiG2F4KXDcbspJJivue1WmVH_w7lwP9xmN0y7-SD1l21WqJvM_yV2_y12_y43bHU0EPSEcfBTN1AIsNbDHA0cEr4DUSJFXMjFZTxiBCWEuHfuAsUqsPz4EIM5TX9z0v7OergWcrMBXEHRMjezIFKblxh88NqbS2OP4e_wLSaRWGrUDAAA=